app/frontend/src/features/communities/discussions/Comment.tsx

```diff
@@ -0,0 +1,122 @@
+import { Card, CircularProgress, Typography } from "@material-ui/core";
+import { Skeleton } from "@material-ui/lab";
+import Avatar from "components/Avatar";
+import Markdown from "components/Markdown";
+import { useUser } from "features/userQueries/useUsers";
+import { Reply } from "pb/threads_pb";
+import { timestamp2Date } from "utils/date";
+import hasAtLeastOnePage from "utils/hasAtLeastOnePage";
+import makeStyles from "utils/makeStyles";
+import { timeAgo } from "utils/timeAgo";
+
+import { getByCreator, UNKNOWN_USER } from "../constants";
+import { useThread } from "../hooks";
+import CommentForm from "./CommentForm";
+
+const useStyles = makeStyles((theme) => ({
+  commentContainer: {
+    alignItems: "start",
+    columnGap: theme.spacing(2),
+    display: "grid",
+    gridTemplateAreas: `
+      "avatar content"
+      "commentForm commentForm"
+    `,
+    gridTemplateColumns: "3rem 1fr",
+    gridTemplateRows: "auto",
+    padding: theme.spacing(2),
+    width: "100%",
+  },
+  commentContent: {
+    "& > * + *": {
+      marginBlockStart: theme.spacing(0.5),
+    },
+    display: "flex",
+    flexDirection: "column",
+    gridArea: "content",
+    marginInlineStart: theme.spacing(1),
+  },
+  avatar: {
+    height: "3rem",
+    gridArea: "avatar",
+    width: "3rem",
+  },
+  nestedCommentsContainer: {
+    "& > * + *": {
+      marginBlockStart: theme.spacing(2),
+    },
+    marginBlockStart: theme.spacing(2),
+    marginInlineStart: theme.spacing(5),
```

Comment:
How about
```suggestion
    marginInlineStart: theme.spacing(3),
    marginInlineStart: `clamp(${theme.spacing(2)}, 5vw, ${theme.spacing(5)})`,
```
Mobile is most likely not to support clamp, so I made the fallback a bit less

Reply:
That's a nice trick actually rather than needing media queries. Still had to do an annoying workaround with '&' to select the same thing again and put in the clamp since you can't have duplicate object keys :(